deployment/helm/templates/agent.yaml

```diff
@@ -17,10 +17,7 @@ spec:
       hostNetwork: true
       dnsPolicy: ClusterFirstWithHostNet
       {{- end }}
-      {{- if .Values.agent.linuxOnly }}
-      nodeSelector:
-        "kubernetes.io/os": linux
-      {{- end }}
+      nodeSelector: {{ include "akri.agent.nodeSelectors" . | nindent 8 }}
```

Comment:
I think we still want to maintain the linux-only node selector switch. Can you mimic the [format of the controller deplpoment](https://github.com/project-akri/akri/blob/main/deployment/helm/templates/controller.yaml), adding the linux only selector if specified and then the other selectors?